tests/foreman/ui/test_myaccount.py

```diff
@@ -111,9 +86,14 @@ def test_positive_update_firstname(self):
 
         :CaseImportance: Critical
         """
-        valid_strs = _valid_string_data()
-        valid_strs.append('name with space')
-        self.assert_text_field_update('users.firstname', *valid_strs)
+        valid_names = _valid_string_data()
+        valid_names.append('name with space')
```

Comment:
No, name with space was on docstring as a valid input, prior to development. So I just added it.